rspace/src/main/scala/coop/rchain/rspace/serializers/ScodecSerialize.scala

```diff
@@ -37,6 +37,12 @@ object ScodecSerialize {
     encodeSortedSeq[Datum[A]](datums, codec)
   }
 
+  def encodeDatum[A](datum: Datum[A])(implicit sa: Serialize[A]): ByteVector = {
+    val codec = serializeToCodecDatumMemo(sa)
+
+    codec.encode(datum).getUnsafe.toByteVector
+  }
```

Comment:
Used in `RholangMergingLogic` to encode single datum which is carried in `ChannelChange[ByteVector]` added/removed lists.